core-services/prow/02_config/openshift/cluster-api-provider-agent/_prowconfig.yaml

```diff
@@ -0,0 +1,111 @@
+tide:
+  queries:
+  - includedBranches:
+    - community-4.6
+    - community-4.7
+    - openshift-4.1
+    - openshift-4.2
+    - openshift-4.3
+    - openshift-4.4
+    - openshift-4.5
+    - openshift-4.6
+    - openshift-4.7
+    - openshift-4.8
+    - openshift-4.9
+    - release-4.0
+    - release-4.1
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    - release-4.7
+    - release-4.8
+    - release-4.9
```

Comment:
all those branches don't exist in this repo, so it's a better idea to clean them up
also the ``bugzilla/valid-bug`` labels, I'm not sure it will work so I think it's better add them when needed